health-services/plan-service/pom.xml

```diff
@@ -4,7 +4,7 @@
     <artifactId>plan-service</artifactId>
     <packaging>jar</packaging>
     <name>plan-service</name>
-    <version>1.0.0</version>
```

Comment:
keep this as 1.0.0 only since this has never been released

Reply:
okay